Source/WebCore/page/PageColorSampler.cpp

```diff
@@ -306,6 +308,11 @@ Color PageColorSampler::predominantColor(Page& page, const LayoutRect& absoluteR
     if (!pixelBuffer)
         return { };
 
+    auto isNearlyTransparent = [&](const Color& color) {
```

Comment:
no need to capture anything
```suggestion
    auto isNearlyTransparent = [](const Color& color) {
```

Reply:
Fixed — thanks for the review!